packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/Relations/RelationModal.tsx

```diff
@@ -33,8 +39,18 @@ import { FormLayout } from '../../FormLayout';
 import type { ContentManagerPlugin, DocumentActionProps } from '../../../../../content-manager';
 
 interface RelationModalProps {
-  open: boolean;
+  triggerButtonLabel: string;
+  newDocument: {
+    documentId: string;
+    model: string;
+    collectionType: string;
+    params: Record<string, string | null>;
+  };
+  setIsConfirmationOpen: (isOpen: boolean) => void;
+  setOnConfirm: (onConfirm: () => void) => void;
+  open?: boolean;
   onToggle: () => void;
+  setIsModalOpen: (isOpen: boolean) => void;
```

Comment:
I think the only props we need here now are `newDocument` and `triggerButtonLabel`.. right? Could you please review the props